samples/highcharts/demo/polar-advanced/demo.html

```diff
@@ -0,0 +1,104 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/coloraxis.js"></script>
+<script src="https://code.highcharts.com/modules/pattern-fill.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<!-- <script src="https://code.highcharts.com/themes/brand-light.js"></script> -->
+
+<figure class="highcharts-figure center">
+    <div id="container"></div>
+    <p class="highcharts-description">
+        An advanced polar chart, showing the performance of three sales teams throughout a month
+    </p>
+</figure>
+
+<pre id="data" style="display: none">
```

Comment:
This inline style throws an error in utils demo view (similar works fine in JSFiddle so I do not think we should care) and importantly, CSS already sets this style so a duplicate.